tests/e2e/Adapter/Scopes/RelationshipTests.php

```
@@ -27,6 +27,214 @@ trait RelationshipTests
+        );
+
+        $zoo1 = $database->createDocument('zoo', new Document([
+            '$id' => 'zoo1',
+            '$permissions' => [
+                Permission::read(Role::any()),
+                Permission::update(Role::any()),
+            ],
+            'name' => 'Bronx Zoo'
+        ]));
+
+        $animal1 = $database->createDocument('animals', new Document([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$animal1'. (Unused Code Rules)

(UnusedLocalVariable)